data/The Heroes.txt

```diff
@@ -482,3 +482,76 @@ person "That Guy"
 		explode "big explosion" 40
 		explode "huge explosion" 50
 		description "Korath World-Ships are mobile habitats cobbled together from asteroids."
+
+#//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+person "Storm Crow"
+	government "Ascendancy"
+	frequency 1000
+	personality
+		plunders
+	phrase
+		word
+			"Storm Crow descending, winter unending. Storm Crow departing, summer is starting."
+	ship "Kar Ik Vot 349"

```

Comment:
Though only mentioned in your person definition, just this will change the default 349. Add a variant name like "349 (Cool Guy)".